docker/pravega/entrypoint.sh

```diff
@@ -9,6 +9,8 @@
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
 
+set -e
```

Comment:
There are two possibilities with a dockerfile. Either keep on running in case of error or exit the startup so that container can be rescheduled somewhere else by the deploying tech. Looks like we are taking the second route. Can you comment more about the decision to go with this option?

Reply:
My reasoning is that we might miss fatal errors if we keep running in case of error. This has just happened with the Python dependency and might happen again. It's much safer to fail fast instead of ignoring the error and running in a potentially degraded mode.